AsyncDisplayKit/ASCellNode.h

```diff
@@ -15,12 +15,13 @@ typedef NSUInteger ASCellNodeAnimation;
 @protocol ASCellNodeLayoutDelegate <NSObject>
 
 /**
- * Notifies the delegate that the specified cell node has done a relayout.
+ * Notifies the delegate that the specified cell node has done a relayout
+ * that resulted in a change of `calculatedSize`.
  * The notification is done on main thread.
  *
  * @param node A node informing the delegate about the relayout.
  */
-- (void)nodeDidRelayout:(ASCellNode *)node;
+- (void)nodeDidRelayoutWithSizeChange:(ASCellNode *)node;
```

Comment:
I would suggest leaving this naming as `didRelayoutNode:`, as it can be used by other future layout invalidations other than size changes. If you need specifics for your current usage of the API, might I suggest adding a boolean flag: `didRelayoutNode:sizeChange:`.